### ISSUE TITLE
Organization: Delete old playground orgs

### ISSUE THREAD

**Issue by garyhtou** [hackclub/hcb#5332]
I'd love to delete old playground orgs that are just hanging around doing nothing. These orgs likely exist because they applied, but were rejected or decided to not onboard.

I'm thinking we delete playground orgs older than 1 year.

@hackliv, is this okay to do from an operations perspective?

---

For engineering, maybe we consider a soft delete (although that can be tricky with associations 🤔)

**Comment by garyhtou** [comment]
Let's hold off on any coding until we determine we plan

**Comment by hackliv** [comment]
yep! I'd say similar to gw, every 6 months the unused ones could get removed.

**Comment by sampoder** [comment]
Events already have `acts_as_paranoid` on them, could we just make a weekly job that goes through and soft deletes things?

**Comment by sampoder** [comment]
@jaspermayone

**Comment by jaspermayone** [comment]
Ty sam!

**Comment by jaspermayone** [comment]
We also can probably delete playground orgs with no organizers on them.

**Comment by jaspermayone** [comment]
Will take a stab at patching this up when I have time!
